plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftCastPushdown.java

```diff
@@ -393,6 +433,41 @@ protected List<CastTestCase> supportedCastTypePushdown()
                 new CastTestCase("c_numeric_30_2", "integer", "c_integer"),
                 new CastTestCase("c_decimal_negative", "integer", "c_integer"),
 
+                new CastTestCase("c_char_10", "char(50)", "c_char_50"),
+                new CastTestCase("c_char_10", "char(256)", "c_bpchar"),
```

Comment:
There are some duplicate test cases. Could you remove them? For instance, `new CastTestCase("c_char_10", "char(256)", "c_bpchar")` exists here and L449.

Reply:
thanks. removed